reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
-
-		var percent =(this.props.score / this.props.questions.length*100);
-		if (percent > 80){
-
-			var message ='Awesome Job!';
-		} else if(percent < 80 && percent >60)
-		{
-			var message = 'You did Ok !';
-		}
-		else {
-
-			var message ='Not satisfactory. Should try once again !';
+		const percent = (this.props.score / this.props.questions.length * 100 * 100);
+		let message;
```

Comment:
You can simplify the logic for determining the message by using a single `if` statement with multiple `else if` conditions. 

Also, it seems like you are multiplying the percentage by 100 twice. This will result in incorrect percentage values.